projects/packages/videopress/src/client/admin/components/video-card/index.tsx

```diff
@@ -164,11 +164,14 @@ export const VideoCard: React.FC< VideoCardProps & VideoThumbnailProps > = ( {
 	onUpdateUpdatePrivacyClick,
 	onDeleteClick,
 } ) => {
-	const playsCount = sprintf(
-		/* translators: placeholder is a product name */
-		__( '%s plays', 'jetpack-videopress-pkg' ),
-		numberFormat( plays )
-	);
+	const hasPlays = typeof plays !== 'undefined';
```

Comment:
`plays` is not present in the ajax endpoint. https://github.com/Automattic/jetpack/issues/26080